main.go

```diff
@@ -156,43 +242,47 @@ func setupChecks(mgr ctrl.Manager) {
 	}
 }
 
-func setupReconcilers(mgr ctrl.Manager) {
-	if webhookPort != 0 {
+func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
+	if webhookPort != 9443 {
```

Comment:
If I'm reading this correctly, a non-default port for the webhookPort will mean the controller isn't created. Is this what you intended?

This section of the v1alpha3 -> v1alpha4 doc suggests under 'Changes in the `main.go` file' suggests all controllers should start no matter what webhook port is selected. 

https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#required-kustomize-changes-to-have-a-single-manager-watching-all-namespaces-and-answer-to-webhook-calls

Reply:
Actually you are right, we do not need to check for it even in `[setupWebhooks](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/main.go#L212)` most probably.